improver_tests/developer_tools/test_MOMetadataInterpreter.py

```diff
@@ -509,6 +564,30 @@ def test_error_missing_spot_coords(blended_spot_median_cube, interpreter):
         interpreter.run(blended_spot_median_cube)
 
 
+@pytest.mark.parametrize(
+    "coord_name", [x for x in SPOT_COORDS if x not in ["latitude", "longitude"]]
+)
+def test_error_inconsistent_spot_coords(
+    blended_spot_median_cube, interpreter, coord_name
+):
+    """Test error raised if a spot cube coord ought to apply to all spots and doesn't"""
```

Comment:
Maybe for clarity this should be `Test error raised if a spot cube coord doesn't meet the expectations of sharing the same dimensions as the x and y coords`?

Reply:
I agree, so I've changed this - not quite the same words you suggested though.